pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -51,6 +51,7 @@ func NewBootstrapParameters() *BootstrapParameters {
 // If the prefix provided doesn't have a "-" then one is added, this makes the
 // generated environment names nicer to read.
 func (io *BootstrapParameters) Complete(name string, cmd *cobra.Command, args []string) error {
+	// io.GitOpsRepoURL = ui.EnterGitOpsRepoURL()
```

Comment:
There are few commented lines in this PR. Are we using them?

Reply:
removed the commented lines which were not required